actions.js

```diff
@@ -1212,20 +1212,43 @@ export function getActions() {
 				choices: this.sourceChoices,
 				isVisible: (options) => options.source === 'custom',
 			},
+			{
+				type: 'checkbox',
+				label: 'Use Custom Path and Name',
+				default: false,
+				id: 'customname',
```

Comment:
This is a bit nit picky, but could we possible use camelCase for this, so `customName`